ERCS/erc-7535.md

```diff
@@ -4,8 +4,7 @@ title: Native Asset ERC-4626 Tokenized Vault
 description: ERC-4626 Tokenized Vaults with Ether (Native Asset) as the underlying asset
```

Comment:
Description `ERC-4626` does not match `eip-7535` title as well as discussion link pointing.